File: Slides/slide7-2.c
#include <stdio.h>

float calcula_media(float * notas){
    float soma = 0;

    for(int i = 0; i < 4; i++){
        soma += notas[i];
    }

    return soma / 4;
}

int main(){

    float notas[4];

    for(int i = 0; i < 4; i++){
        scanf("%f", &notas[i]);
    }

    float media = calcula_media(notas);

    printf("Media = %.2f\n", media);

    return 0;
}

File: Slides/slide7-2.py
def calcula_media(notas):
    soma = 0

    for n in notas:
        soma += n
    
    return soma / 4


notas = [0.0]*4

for i in range(len(notas)):
    notas[i] = float(input())

media = calcula_media(notas)

print(f"Media = {media:.2f}")